test/nuxeo.spec.js

```diff
@@ -366,5 +367,25 @@ describe('Nuxeo', () => {
       const options = nuxeo._computeFetchOptions(defaultOptions);
       expect(options.url).to.be.equal('http://localhost:8080/nuxeo?param=1&param=2&param=3');
     });
+
+    it('should keep the abort signal property if it is provided', () => {
+      const defaultOptions = {
+        url: 'http://localhost:8080/nuxeo',
+        signal: new AbortController().signal,
```

Comment:
Just use a dummy object here as you just want to validate it's the same object in the fetch options.

Reply:
No need to add a useless dependency, dummy object here will do the job with less maintenance.

If you give a wrong object, either `fetch` will crash or do nothing, that's the responsibility of the caller. We do not check the type of other options either.